pkg/utils/utils.go

```diff
@@ -81,6 +82,20 @@ func GetPodIdentifier(podName, podNamespace string) string {
 	return podIdentifierPrefix + "-" + podNamespace
 }
 
+func addDots(podName string) string {
+	var dotString strings.Builder
+
+	for _, c := range podName {
+		if c == '.' {
+			dotString.WriteString("dot")
```

Comment:
I think replacing with the string "dot" makes this harder to read. Can we just replace with "_" instead? I guess we have to be careful about conflicting with existing pod names, but I do not see a way around that

Reply:
I see, yeah in upcoming revision it is "_"